### PR TITLE
Falkreath Hauntings - Add Do not clean message

### PR DESCRIPTION
**Pull request by TateTaylorOH**
opusGlass' [Falkreath Hauntings](https://www.nexusmods.com/skyrimspecialedition/mods/20733/) contains 3 ITMs. These are required in order to for the Lady Ilinalta encounter to work properly. They are edits to flow data so that the freezing effect that the boss battle happens as is scripted. Water mods and any other mod that edits flow data in that location will cause the water not to freeze and render the boss battle impossible to complete.

### CODE REVIEW

**Comment by MacSplody** [comment]
Cleaning won't remove the changes to water flow, it does remove two vanilla navmesh though.

image

There is a way to keep those changes from being cleaned by changing the editorID.
But I think It's probably a conflict with another mod, and given the area it's in, that could be a lot of mods.
Moon and Star, Undeath, Tel Nalta, Lake Haven, Millwater Retreat, Oar's Rest to name a few.

image

**Comment by TateTaylorOH** [comment]
Changing the editiorID is a good idea.

Yeah, there's a lot of patches available, could we also get a LOOT warning if those patches aren't detected?

File: masterlist.yaml [hunk 1]
```diff
@@ -5287,11 +5287,10 @@ plugins:
...
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/20733/' ]
-    dirty:
-      - <<: *quickClean
-        crc: 0xF302A166
-        util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
-        itm: 3
+    msg:
+      - type: say
+        content: 'Do not clean. "Dirty" edits are intentional and required for the mod to function.'
+        lang: eng
```
Comment (MacSplody) [inline masterlist.yaml]: You could use the alias doNotClean instead, it does have a slightly different message.

> Do not clean ITM records, they are intentional and required for the mod to function. It is safe to undelete records, but do not do anything other than that.

```suggestion
    msg: [ *doNotClean ]
```

Comment (TateTaylorOH) [reply masterlist.yaml]: That works for me if it works for you.

**Comment by MacSplody** [comment]
> Changing the editiorID is a good idea.
> 
> Yeah, there's a lot of patches available, could we also get a LOOT warning if those patches aren't detected?

Adding those is pretty straight forward, there are a few different alias types depending on where the patch is hosted.

For example there's the alias `patchProvided` for any patches on the original modpage.
```yaml
  - name: 'Tel_Nalta.esp'
    msg:
      - <<: *patchProvided
        subs: [ 'Moon and Star' ]
        condition: 'active("MoonAndStar_MAS.esp") and not active("MoonAndStar_TelNaltaPatch.esp")'
```

**Comment by TateTaylorOH** [comment]
They should all be hosted on the mod page.

**Comment by MacSplody** [comment]
@PierreDespereaux Thanks.

**Comment by TateTaylorOH** [comment]
Thank _you_!

**Pull request merged**